dashboard/app/controllers/courses_controller.rb

```diff
@@ -112,27 +112,31 @@ def vocab
     raise ActiveRecord::RecordNotFound unless unit_group
     # Assumes if one unit in a unit group is migrated they all are
     return render :forbidden unless unit_group.default_scripts[0].is_migrated
+    render 'vocab', locals: {course_summary: unit_group.summarize_for_rollup(@current_user)}
```

Comment:
I don't see this pattern very often in our codebase. please set a variable like this instead:
```
@course_summary = unit_group.summarize_for_rollup(@current_user)
```
and then access `@course_summary` from the haml file. then you will not need to explicitly `render 'vocab'`. same for other controller methods in this file.